WebRoot/js/zimbraMail/share/model/ZmSearch.js

```diff
@@ -421,7 +421,7 @@ function(params) {
 					if (this.types.contains(ZmItem.MSG) || this.types.contains(ZmItem.CONV)) {
 						// special handling for showing participants ("To" instead of "From")
 						var folder = this.folderId && appCtxt.getById(this.folderId);
-						request.recip = (folder && folder.isOutbound()) ? "2" : "0";
+						request.recip = (folder && folder.isOutbound()) ? this.types.contains(ZmItem.MSG) ? "3" : "2" : "0";
```

Comment:
this might be more readable...?
```suggestion
						request.recip = (folder && folder.isOutbound()) ? (this.types.contains(ZmItem.MSG) ? "3" : "2") : "0";
```

Reply:
@k-kato  Done.